domains/esb.is-cool.dev.json

```diff
@@ -0,0 +1,17 @@
+{
+    "description": "Minecraft Bot Page",
+
+    "domain": "is-cool.dev",
+    "subdomain": "esb",
+
+    "owner": {
+        "repo": "https://github.com/EducatedSuddenBucket/Ferrum",
+        "email": "educatedsuddenbucket@gmail.com"
+    },
+
+    "record": {
+        "A": ["69.197.135.202"]
```

Comment:
502 Gateway, are you able to provide a site preview?

Reply:
yes here ,https://educatedsuddenbucket.is-a.dev/Ferrum/ (its a preview) not a the actual site

Reply:
also the A record is a reverse proxy